kaspad/src/main.rs

```diff
@@ -188,7 +203,10 @@ pub fn main() {
     // DB used for addresses store and for multi-consensus management
     let meta_db = kaspa_database::prelude::open_db(meta_db_dir, true, 1);
 
-    let grpc_server_addr = args.rpclisten.unwrap_or_else(|| format!("0.0.0.0:{}", config.default_rpc_port())).parse().unwrap();
+    let connect_peers = args.connect_peers.iter().map(|x| x.normalize(config.default_p2p_port())).collect();
+    let add_peers = args.add_peers.iter().map(|x| x.normalize(config.default_p2p_port())).collect();
+    let listen = args.listen.unwrap_or_default().normalize(config.default_rpc_port());
```

Comment:
`default_rpc_port` -> `default_p2p_port`. Also probably better to rename `listen` -> `p2p_server_addr`

Reply:
Nice catch!